packages/ui/icons/comments/small.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { PureComponent } from "react";
+
+
+export default class CommentsSmall extends PureComponent {
+  render() {
+    return (
+      <svg fill="transparent" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
+        <g clipPath="url(#clip0_1367_271)">
+          <path d="M5.91328 13.5316C5.38594 13.3418 4.79883 13.4227 4.34531 13.7566C4.05703 13.9676 3.56133 14.277 2.96016 14.5547C3.15703 14.0379 3.3082 13.4543 3.35742 12.818C3.39258 12.3645 3.24141 11.9145 2.94258 11.5699C2.12344 10.6453 1.6875 9.5625 1.6875 8.4375C1.6875 5.64258 4.61602 2.8125 9 2.8125C13.384 2.8125 16.3125 5.64258 16.3125 8.4375C16.3125 11.2324 13.384 14.0625 9 14.0625C7.88906 14.0625 6.84492 13.8691 5.91328 13.5316ZM0.924609 14.8992C0.868359 14.9941 0.808594 15.0891 0.745313 15.184L0.734766 15.2016C0.678516 15.2824 0.622266 15.3633 0.566016 15.4441C0.442969 15.6094 0.309375 15.7711 0.16875 15.9187C0.00703126 16.0805 -0.0386719 16.3195 0.0492187 16.5305C0.137109 16.7414 0.341016 16.8785 0.569531 16.8785C0.748828 16.8785 0.928125 16.868 1.10742 16.8504L1.13203 16.8469C1.28672 16.8293 1.44141 16.8082 1.59609 16.7801C1.62422 16.7766 1.65234 16.7695 1.68047 16.7625C2.30625 16.6395 2.90742 16.4285 3.4418 16.1965C4.24687 15.8449 4.93242 15.4266 5.35078 15.1207C6.46875 15.525 7.70625 15.75 9.01055 15.75C13.9816 15.75 18.0105 12.477 18.0105 8.4375C18.0105 4.39805 13.9711 1.125 9 1.125C4.02891 1.125 0 4.39805 0 8.4375C0 10.023 0.622266 11.4891 1.67695 12.6879C1.61016 13.5492 1.27617 14.3156 0.924609 14.8992ZM5.0625 9.5625C5.36087 9.5625 5.64702 9.44397 5.858 9.233C6.06897 9.02202 6.1875 8.73587 6.1875 8.4375C6.1875 8.13913 6.06897 7.85298 5.858 7.642C5.64702 7.43103 5.36087 7.3125 5.0625 7.3125C4.76413 7.3125 4.47798 7.43103 4.267 7.642C4.05603 7.85298 3.9375 8.13913 3.9375 8.4375C3.9375 8.73587 4.05603 9.02202 4.267 9.233C4.47798 9.44397 4.76413 9.5625 5.0625 9.5625ZM10.125 8.4375C10.125 8.13913 10.0065 7.85298 9.7955 7.642C9.58452 7.43103 9.29837 7.3125 9 7.3125C8.70163 7.3125 8.41548 7.43103 8.2045 7.642C7.99353 7.85298 7.875 8.13913 7.875 8.4375C7.875 8.73587 7.99353 9.02202 8.2045 9.233C8.41548 9.44397 8.70163 9.5625 9 9.5625C9.29837 9.5625 9.58452 9.44397 9.7955 9.233C10.0065 9.02202 10.125 8.73587 10.125 8.4375ZM12.9375 9.5625C13.2359 9.5625 13.522 9.44397 13.733 9.233C13.944 9.02202 14.0625 8.73587 14.0625 8.4375C14.0625 8.13913 13.944 7.85298 13.733 7.642C13.522 7.43103 13.2359 7.3125 12.9375 7.3125C12.6391 7.3125 12.353 7.43103 12.142 7.642C11.931 7.85298 11.8125 8.13913 11.8125 8.4375C11.8125 8.73587 11.931 9.02202 12.142 9.233C12.353 9.44397 12.6391 9.5625 12.9375 9.5625Z" />
+        </g>
+        <defs>
+          <clipPath id="clip0_1367_271">
+            <rect fill="white" height="18" width="18" />
```

Comment:
@AlexBabusanu is the `white` color in this rectangle supposed to be static ? or should it be able to inherit from the `svg fill color` or `currentColor` ?